sdks/python/apache_beam/transforms/util.py

```diff
@@ -671,6 +679,60 @@ def WithKeys(pcoll, k):
   return pcoll | Map(lambda v: (k, v))
 
 
+@typehints.with_input_types(typehints.KV[K, V])
+class GroupIntoBatches(PTransform):
+  def __init__(self, batch_size):
+    self.batch_size = batch_size
+
+  @staticmethod
+  def of_size(batch_size):
```

Comment:
I do not believe we need `of_size` in Python. Users can directly use GroupIntoBatches().

Reply:
fixed